From f8a757e86cf1cffef455b251425947338198afc4 Mon Sep 17 00:00:00 2001
From: Algernon <lixiaodong67@huawei.com>
Date: Mon, 3 Jul 2023 17:03:00 +0800
Subject: [PATCH 126/268] xdp: Fixed an issue where the trace_mem_disconnect
 function cannot find the definition.

driver inclusion
category: bugfix
bugzilla: https://gitee.com/src-openeuler/kernel/issues/I8EMYT

--------------------------------

Fix kabi changes caused by the introduction of new xdp header files.

Signed-off-by: Xiaodong Li <lixiaodong67@huawei.com>
---
 net/core/xdp.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/net/core/xdp.c b/net/core/xdp.c
index 89b6785cef2a..978bd4432a73 100644
--- a/net/core/xdp.c
+++ b/net/core/xdp.c
@@ -14,6 +14,9 @@
 #include <net/page_pool.h>
 
 #include <net/xdp.h>
+#ifndef __GENKSYMS__
+#include <trace/events/xdp.h>
+#endif
 
 #define REG_STATE_NEW		0x0
 #define REG_STATE_REGISTERED	0x1
-- 
2.34.1

